### PR TITLE
[TF] optimization continued

### PR DESCRIPTION
**Pull request by abhinavsp0730**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.
Notebook covered
1. convexity
2. gd
3. sgd

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1262/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1262/

File: chapter_optimization/sgd.md [hunk 1]
```diff
@@ -44,7 +53,7 @@ This means that, on average, the stochastic gradient is a good estimate of the g
 Now, we will compare it to gradient descent by adding random noise with a mean of 0 and a variance of 1 to the gradient to simulate a SGD.
 
 ```{.python .input}
-#@tab all
+# removed tab all because code isn't same for tf
```
Comment (terrytangyuan) [inline chapter_optimization/sgd.md:56]: No need to add comment here

File: chapter_optimization/sgd.md [hunk 2]
```diff
@@ -62,6 +71,45 @@ lr = (lambda: 1)  # Constant learning rate
 d2l.show_trace_2d(f, d2l.train_2d(sgd, steps=50))
 ```
 
+```{.python .input}
+#@tab pytorch
+# removed tab all because code isn't same for tf
```
Comment (terrytangyuan) [inline chapter_optimization/sgd.md:76]: No need to comment here in PyTorch tab

**Comment by abhinavsp0730** [comment]
Actually I've added because previously it's under #@tab all. Should I
removed the comments or It'll removed when doing the polishing?

**Comment by terrytangyuan** [comment]
I'll leave that to @mli @astonzhang to comment.

**Comment by mli** [comment]
Job d2l-en/PR-1262/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1262/

**Comment by abhinavsp0730** [comment]
I guess the comment is unnecessary. Yuan please review now.
